app/src/components/layout/Home/PeersValidatorsInfo/PeersValidatorsInfo.tsx

```diff
@@ -40,22 +56,36 @@ export const PeersValidatorsInfo: React.FC<PeersValidatorsInfoProps> = ({
             <PageLink to="/validators">{t('view-all')}</PageLink>
           </TextWrapper>
           <H3Data>
-            {standardizeNumber(currentEraValidatorStatus?.validatorsCount ?? 0)}
+            {isLoadingValidators ? (
+              <Skeleton width={60} duration={1} />
+            ) : (
+              standardizeNumber(currentEraValidatorStatus?.validatorsCount ?? 0)
+            )}
           </H3Data>
           <DataContext>
-            {t('active-bids', {
-              activeBids: standardizeNumber(
-                currentEraValidatorStatus?.bidsCount ?? 0,
-              ),
-            })}
+            {isLoadingValidators ? (
+              <Skeleton width="70%" duration={1} />
```

Comment:
this `width` values are specific to this certain view, right? in future when we'll have a few views that will use the same value, I would add them to the `constants` to get rid of all of the `magic strings` from the code :)

Reply:
Yes! I should have left a comment - when we create skeleton loaders for all the tables I plan on creating a reusable component that will have set `options` that can be passed in :)